openhcl/underhill_core/src/worker.rs

```diff
@@ -2553,8 +2541,6 @@ async fn new_underhill_vm(
             } else {
                 TpmAkCertTypeResource::Trusted(request_ak_cert)
             }
-        } else {
-            TpmAkCertTypeResource::None
```

Comment:
>TpmAkCertTypeResource::None [](http://example.com/codeflow?start=12&length=27)

do we still want this type going forward?  In what case do we want this logic?

Reply:
We still use None in openvmm_entry and pertri environments. Not sure what does it take to using non-None there.